spring-pulsar-sample-apps/build.gradle

```diff
@@ -8,6 +8,12 @@ description = 'Spring Pulsar Sample Applications'
 dependencies {
 	api project(':spring-pulsar-spring-boot-starter')
 	implementation 'com.google.code.findbugs:jsr305'
+
+	// observability
```

Comment:
I am going to add a section in the sample apps readme as well that shows how to run Zipkin locally. I may pull some of these doc details from the main docs to the sample app readme as they are a bit detailed currently.